src/main/resources/application.yaml

```diff
@@ -41,7 +41,7 @@ logging:
 
 okta:
   oauth2:
-    issuer: ${OKTA_ISSUER:https://dev-18092578.okta.com/oauth2/default}
```

Comment:
do we need to change the default okta instance?

Reply:
that was a mistake @nmorasb I reverted it. Thanks